flexget/plugins/metainfo/imdb_lookup.py

```diff
@@ -52,6 +58,7 @@ class Movie(Base):
     genres = relation('Genre', secondary=genres_table, backref='movies')
     actors = relation('Actor', secondary=actors_table, backref='movies')
     directors = relation('Director', secondary=directors_table, backref='movies')
+    writers = relation('Writer', secondary=writers_table, backref='movies')
```

Comment:
When you change the DB Schema, you need to bump the Base version and update the upgrade method.